src/DependencyInjection/SonataUserExtension.php

```diff
@@ -178,7 +178,15 @@ public function configureGoogleAuthenticator($config, ContainerBuilder $containe
         }
 
         $container->setParameter('sonata.user.google.authenticator.forced_for_role', $config['google_authenticator']['forced_for_role']);
-        $container->setParameter('sonata.user.google.authenticator.ip_white_list', $config['google_authenticator']['ip_white_list']);
+
+        if (array_key_exists('ip_white_list', $config['google_authenticator']) && array_key_exists('trusted_ip_list', $config['google_authenticator'])) {
+            throw new \RuntimeException('Please use only ``trusted_ip_list`` parameter. ``ip_white_list`` is deprecated.');
```

Comment:
This should be fixed by changing code, so let's use `\LogicException` here
```suggestion
            throw new \LogicException('Please use only ``trusted_ip_list`` parameter. ``ip_white_list`` is deprecated.');
```

Reply:
Also, why the double backticks?

Reply:
Seen in line 177 of the same class, I've tried to keep same notation (don't know if must be single or double backtick)

Reply:
This is neither markdown nor rst, don't know why it was contributed like this. IMO we should use double quotes…